<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MntConfigWidget</class>
 <widget class="QWidget" name="MntConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>350</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="mGroupBox">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="iconNameLabel">
        <property name="text">
         <string>Icon name:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="mountLabel">
        <property name="text">
         <string>Mount Command:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="umountLabel">
        <property name="text">
         <string>Unmount Command:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="mDefaultIconButton">
        <property name="text">
         <string>Default Icon</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="mMountButton">
        <property name="text">
         <string>Get Command...</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="mUmountButton">
        <property name="text">
         <string>Get Command...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="KIconButton" name="mIconButton"/>
      </item>
      <item row="1" column="1">
       <widget class="KLineEdit" name="mIconLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="KLineEdit" name="mMountLineEdit"/>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="KLineEdit" name="mUmountLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTreeWidget" name="m_listWidget">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIconButton</class>
   <extends>QPushButton</extends>
   <header>kicondialog.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
